relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -27,11 +27,12 @@ const PLACEHOLDER_UNPARAMETERIZED: &str = "<< unparameterized >>";
 /// Tags we set on metrics for performance score measurements (e.g. `score.lcp.weight`).
 ///
 /// These are a subset of "universal" tags.
-const PERFORMANCE_SCORE_TAGS: [CommonTag; 6] = [
+const PERFORMANCE_SCORE_TAGS: [CommonTag; 7] = [
     CommonTag::BrowserName,
     CommonTag::Environment,
     CommonTag::GeoCountryCode,
     CommonTag::Release,
+    CommonTag::ScoreProfileVersion,
```

Comment:
I'm not entirely sure if this should be considered a `CommonTag`. It's not used for every transaction. What is the definition of `"universal" tags` being used here? Is there a better alternative to define this if I want to isolate this tag from the universal ones?

Reply:
It would be slightly better to declare this as an optional field on `TransactionMeasurementTags`, just like `measurement_rating`. You then just need to make sure that it gets converted in https://github.com/getsentry/relay/blob/154516c08de16df0aef96e29c449e3f5eefb6d26/relay-server/src/metrics_extraction/transactions/types.rs#L183

Reply:
This is great, thanks! Added.